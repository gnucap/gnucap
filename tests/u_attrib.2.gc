
option list
verilog
module aaa (c,d);
	resistor #(100) r1(a,b);	
	resistor #((*r*)200,(*m*)2) r2(a,b);
	r1 r3(a,b);
	r2 r4(a,b);
	r1 r5(a,b);
	r2 r6(a,b);
endmodule;

module bbb ((*b*)e,f);
        resistor #((*a*)100) r1(a,b);
	resistor #((*d*).$mfactor(2),.r(200)) r2(a,b);
	resistor #(.$mfactor(2),(*d*).r(200)) r2a(a,b);
	r1 r3(a,b);
	r2 r4(a,b);
	r1 r5(a,b);
	r2 r6(a,b);
	aaa a1(a,b);
	aaa a2(.d(a),.c(b));
	aaa a3(a,(*u*)b);
	aaa a4((*u*).d(a),.c(b));
endmodule;
	
list

end
