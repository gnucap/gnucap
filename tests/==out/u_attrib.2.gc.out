"verilog "
"module aaa (c,d); "
"resistor #(100) r1(a,b); "
"resistor #((*r*)200,(*m*)2) r2(a,b); "
"r1 r3(a,b); "
"r2 r4(a,b); "
"r1 r5(a,b); "
"r2 r6(a,b); "
"endmodule; "
"module bbb ((*b*)e,f); "
"resistor #((*a*)100) r1(a,b); "
"resistor #((*d*).$mfactor(2),.r(200)) r2(a,b); "
"resistor #(.$mfactor(2),(*d*).r(200)) r2a(a,b); "
"r1 r3(a,b); "
"r2 r4(a,b); "
"r1 r5(a,b); "
"r2 r6(a,b); "
"aaa a1(a,b); "
"aaa a2(.d(a),.c(b)); "
"aaa a3(a,(*u*)b); "
"aaa a4((*u*).d(a),.c(b)); "
"endmodule; "
"list "
module aaa (c,d);
resistor #(.r(100)) r1 (.p(a),.n(b));
resistor #((* r *) .r(200),(* m *) .$mfactor(2)) r2 (.p(a),.n(b));
resistor #(.r(100)) r3 (.p(a),.n(b));
resistor #(.r(200),.$mfactor(2)) r4 (.p(a),.n(b));
resistor #(.r(100)) r5 (.p(a),.n(b));
resistor #(.r(200),.$mfactor(2)) r6 (.p(a),.n(b));
endmodule // aaa

module bbb ((* b *) e,f);
resistor #((* a *) .r(100)) r1 (.p(a),.n(b));
resistor #(.r(200),(* d *) .$mfactor(2)) r2 (.p(a),.n(b));
resistor #((* d *) .r(200),.$mfactor(2)) r2a (.p(a),.n(b));
resistor #(.r(100)) r3 (.p(a),.n(b));
resistor #(.r(200),.$mfactor(2)) r4 (.p(a),.n(b));
resistor #(.r(100)) r5 (.p(a),.n(b));
resistor #(.r(200),.$mfactor(2)) r6 (.p(a),.n(b));
aaa #() a1 (.c(a),.d(b));
aaa #() a2 (.c(b),.d(a));
aaa #() a3 (.c(a),(* u *) .d(b));
aaa #() a4 (.c(b),(* u *) .d(a));
endmodule // bbb

"end "
